docker/Dockerfile.ruby_host

```diff
@@ -0,0 +1,23 @@
+FROM ruby:2.6-alpine as build
+
+ENV BUNDLER_VERSION 2.1.4
+ENV LANG=C.UTF-8 \
+    BUNDLE_JOBS=6 \
+    BUNDLE_RETRY=3
+
+RUN apk add --no-cache --update build-base git jq curl
+
+RUN mkdir -p /app
+COPY Gemfile* /app/
+WORKDIR /app
+
+RUN gem install bundler -v $BUNDLER_VERSION && bundle install --with test && gem cleanup
+
+RUN addgroup --gid 1000 ruby && \
+    adduser --home /home/ruby --uid 1000 --ingroup ruby --disabled-password ruby
+
+COPY . /app
+
+RUN rm ./testing/artifacts/*/.keep && chown -R 1000:1000 /app
```

Comment:
Do we need to remove these files as they preserve the folder structure (Or will they still preserve structure after removing them?)

Reply:
They're only removed from the folder structure inside the container after the copy, so the mount point is empty and should allow writes properly from the non-root processes inside the container.